.github/workflows/backend-ci.yml

```
@@ -0,0 +1,108 @@
+        retention-days: 7
+
+  security:
+    runs-on: ubuntu-latest
+    needs: test
+    
+    defaults:
+      run:
+        working-directory: ./backend
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)